### PR TITLE
III-4284 Refactor get organizer

### PR DESCRIPTION
**Pull request by LucWollants**
### Added
- Added new `GetOrganizerRequestHandler`

### Removed
- Removed obsolete `ReadOrganizerRestController`

---
Ticket: https://jira.uitdatabank.be/browse/III-4284

### CODE REVIEW

File: src/Http/Organizer/GetOrganizerRequestHandler.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        try {
+            $organizer = $this->organizerService->getEntity($organizerId);
+
+            return (new JsonLdResponse())->withBody(
+                (new StreamFactory())->createStream($organizer)
+            );
+        } catch (EntityNotFoundException $exception) {
+        }
+
+        throw ApiProblem::organizerNotFound($organizerId);
```
Comment (jdrieghe) [inline src/Http/Organizer/GetOrganizerRequestHandler.php:40]: Should this live inside the `catch`?

File: src/Http/Organizer/GetOrganizerRequestHandler.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    {
+        $routeParameters = new RouteParameters($request);
+        $organizerId = $routeParameters->getOrganizerId();
+
+        try {
+            $organizer = $this->organizerService->getEntity($organizerId);
+
+            return (new JsonLdResponse())->withBody(
+                (new StreamFactory())->createStream($organizer)
+            );
```
Comment (jdrieghe) [inline src/Http/Organizer/GetOrganizerRequestHandler.php:36]: I wonder if there's a less convoluted way to construct this. Maybe we should build custom tooling for it?

**Review by jdrieghe — APPROVED** [review]
Some questions, nothing blocking. Nice work @LucWollants

File: src/Http/Organizer/GetOrganizerRequestHandler.php (hunk 2 above)
Comment (LucWollants) [reply src/Http/Organizer/GetOrganizerRequestHandler.php]: I changed the improved the `JsonResponse` constructor and it now only does a `json_encode` on types different from `string`.

**Review by jdrieghe — APPROVED** [review]

**Pull request merged**